HTTPServer.test/ServerTests.cs

```diff
@@ -21,23 +25,39 @@ public class ServerUnitTests
         [TestMethod]
         public void ServerCanStart()
         {
-            Assert.True(server.Start() && server.Running);
+            Assert.True(server.Start(8080) != null && server.Running);
```

Comment:
What's the use of the boolean return? Can it just be idempotent so `server.Start().Start().Start()` works and `server.Stop().Stop().Stop()` works?